pkgs/development/python-modules/desktop-entry-lib/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, pytestCheckHook
+, fetchFromGitea
+, gitUpdater
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "desktop-entry-lib";
+  version = "3.1";
+  format = "pyproject";
+  disabled = pythonOlder "3.9";
```

This was changed recently and it the new way to specify the "format".

```suggestion
  pyproject = true;

  disabled = pythonOlder "3.9";
```